File: app/src/main/java/com/jakelauer/baseballtheater/Baseballtheater.java
package com.jakelauer.baseballtheater;

import android.app.Application;

import com.facebook.stetho.Stetho;
import com.facebook.stetho.okhttp3.StethoInterceptor;

import org.joda.time.DateTime;

import java.util.Date;

import okhttp3.OkHttpClient;

/**
 * Created by Jake on 1/21/2017.
 */

public class BaseballTheater extends Application {
	public void onCreate() {
		super.onCreate();
		Stetho.initializeWithDefaults(this);
	}
}

File: app/src/main/java/com/jakelauer/baseballtheater/BaseballTheater.kt
package com.jakelauer.baseballtheater

import android.app.Application
import android.content.SharedPreferences
import android.preference.PreferenceManager
import com.facebook.stetho.Stetho

/**
 * Created by Jake on 1/21/2017.
 */

class BaseballTheater : Application()
{
	lateinit var m_prefs: SharedPreferences
	private set

	override fun onCreate()
	{
		super.onCreate()
		Stetho.initializeWithDefaults(this)

		m_prefs = PreferenceManager.getDefaultSharedPreferences(this)
	}
}